ci/scripts/run-fpga-tests.sh

```diff
@@ -44,6 +44,9 @@ trap 'ci/bazelisk.sh run //sw/host/opentitantool -- --rcfile= --interface=${fpga
 ci/bazelisk.sh run //sw/host/opentitantool -- --rcfile= --interface="$fpga" --logging debug fpga set-pll || true
 ci/bazelisk.sh run //sw/host/opentitantool -- --rcfile= --interface="$fpga" fpga clear-bitstream
 
+# Print the SAM3X firmware version.
+ci/bazlisk.sh run //sw/host/opentitantool -- --rcfile= --interface="$fpga" fpga get-sam3x-fw-version
+
```

Comment:
`bazelisk` is misspelled. Otherwise looks good to me. You could also echo something before printing the version, if the output itself isn't sufficiently self-descriptive.

Reply:
Oops, impressive to typo a one line change. Maybe I should have found a way to run this locally. Thanks.

Bazel prints the command it's running so I think it's fairly self descriptive.